solidity/test/bridge/Bridge.Frauds.test.ts

```diff
@@ -28,20 +28,134 @@ const { lastBlockTime, increaseTime } = helpers.time
 const fixture = async () => bridgeFixture()
 
 describe("Bridge - Fraud", () => {
+  let governance: SignerWithAddress
   let thirdParty: SignerWithAddress
   let treasury: SignerWithAddress
   let bridge: Bridge & BridgeStub
 
-  let fraudChallengeDefeatTimeout
-  let fraudChallengeDepositAmount
+  let fraudChallengeDefeatTimeout: BigNumber
+  let fraudChallengeDepositAmount: BigNumber
 
   before(async () => {
     // eslint-disable-next-line @typescript-eslint/no-extra-semi
-    ;({ thirdParty, treasury, bridge } = await waffle.loadFixture(fixture))
+    ;({ governance, thirdParty, treasury, bridge } = await waffle.loadFixture(
+      fixture
+    ))
     ;({ fraudChallengeDefeatTimeout, fraudChallengeDepositAmount } =
       await bridge.fraudParameters())
   })
 
+  describe("updateFraudParameters", () => {
```

Comment:
Non-blocking this PR, especially that this approach is aligned with `updateWalletParameters` function, but I think we should have a separate test file `Bridge.Parameters.test.ts` with tests for `update*` functions. Let's do it at the very end once we have all already opened PRs merged.

Reply:
Done in the follow-up PR in https://github.com/keep-network/tbtc-v2/pull/229/commits/49f3a5d7a6cabc18a383e4b72c7f4077cdf79499.